practitioner-service/pom.xml

```diff
@@ -3,21 +3,22 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">  
   <modelVersion>4.0.0</modelVersion>  
   <parent> 
-    <groupId>org.springframework.boot</groupId>  
-    <artifactId>spring-boot-starter-parent</artifactId>  
-    <version>2.3.6.RELEASE</version>  
-    <relativePath/>  
+    <groupId>uk.ac.ox.ctsu</groupId>  
+    <artifactId>java-base-app</artifactId>  
+    <version>0.0.1-SNAPSHOT</version>  
+    <relativePath>../</relativePath>  
```

Comment:
Since you fixed the inheritance from the parent POM, then probably should also remove all dependencies here that come from the parent. right?

Reply:
inheritance of POM is not fixed as part of this PR, and the projects still do not work properly in any IDE. 
The POM structure is part of another user story, these are just initial settings which were wrong.